package.json

```diff
@@ -44,18 +44,10 @@
 			"react": "18.3.1",
 			"react-docgen-typescript": "2.2.2",
 			"react-native": "~0.75.0",
-			"react-native-reanimated": "~3.15.5",
-			"@sphereon/did-auth-siop": "https://gitpkg.vercel.app/animo/OID4VC/packages/siop-oid4vp?funke",
-			"@sphereon/oid4vc-common": "https://gitpkg.vercel.app/animo/OID4VC/packages/common?funke",
-			"@sphereon/oid4vci-common": "https://gitpkg.vercel.app/animo/OID4VC/packages/oid4vci-common?funke",
-			"@sphereon/oid4vci-issuer": "https://gitpkg.vercel.app/animo/OID4VC/packages/issuer?funke",
-			"@sphereon/oid4vci-client": "https://gitpkg.vercel.app/animo/OID4VC/packages/client?funke",
-			"@sphereon/jarm": "https://gitpkg.vercel.app/animo/OID4VC/packages/jarm?funke",
-			"@protokoll/mdoc-client": "0.2.36"
+			"react-native-reanimated": "~3.15.5"
 		},
 		"patchedDependencies": {
-			"@hyperledger/indy-vdr-react-native@0.2.2": "patches/@hyperledger__indy-vdr-react-native@0.2.2.patch",
-			"@protokoll/mdoc-client": "patches/@protokoll__mdoc-client.patch"
+			"@hyperledger/indy-vdr-react-native@0.2.2": "patches/@hyperledger__indy-vdr-react-native@0.2.2.patch"
```

Comment:
this patch is needed, otherwise Jan won't be able to do the flow

Reply:
Yes see my question in slack. Can't we make a fix in the actual mdoc repo?